ClangSharp.PInvokeGenerator/Cursors/Attrs/Attr.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using System.Diagnostics;
-using ClangSharp;
 
-namespace ClangSharpPInvokeGenerator
+namespace ClangSharp
```

Comment:
All of the structured traverser is still internal. It might be nice to make it public/reusable now that this is a separate library...